addons/account/static/src/components/section_and_note_fields_backend/section_and_note_fields_backend.js

```diff
@@ -274,20 +327,53 @@ export class SectionAndNoteListRenderer extends ListRenderer {
         return record.data.display_type === DISPLAY_TYPES.SECTION;
     }
 
+    isSubSection(record) {
+        return record.data.display_type === DISPLAY_TYPES.SUBSECTION;
+    }
+
+    hideRecord(record, state) {
```

Comment:
```suggestion
    isLineVisible(record, state) {
    isHidden() or ...
```
or something similar